src/js/components/Tab/Tab.js

```diff
@@ -13,118 +13,107 @@ import { normalizeColor } from '../../utils';
 
 import { StyledTab } from './StyledTab';
 
-class Tab extends Component {
-  static getDerivedStateFromProps(nextProps, prevState) {
-    const { active } = nextProps;
-    const { over } = prevState;
-    if (active && over) {
-      return { over: undefined };
-    }
-    return null;
-  }
-
-  state = {};
-
-  onMouseOver = event => {
-    const { onMouseOver } = this.props;
-    this.setState({ over: true });
+const Tab = props => {
+  const {
+    active,
+    forwardRef,
+    plain,
+    title,
+    onMouseOver,
+    onMouseOut,
+    theme,
+    ...rest
+  } = props;
```

Comment:
can we move the destructure in the function declaration?

Reply:
I believe you're right. Let me try doing that and confirm it still works.